atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -75,6 +77,10 @@ public KeyValueService getKeyValueService() {
         return keyValueService.get();
     }
 
+    public KeyValueServiceConfig getMergedKeyValueServiceConfig() {
```

Comment:
`getKeyValueService` already returns the one true key value service. Having a duplicative method here is unnecessary and error-prone.

Reply:
this is the config, not the service

Reply:
Oh I see, sorry.

Can we expose this through the existing value? It feels wrong to build this config object twice.